### DISCUSSION TITLE
使用vue3+elementplus+vite写后台管理引入unocss不能正常使用官方预定义的属性

### DISCUSSION THREAD

**Discussion by rabbit-lili** [unocss/unocss#3241] [Q&A] [answered]
初始化搭建项目时还都是正常的，按需引入elementplus后，在运行后，unocss的样式全部失效<div class="c-red">文字</div>
![image](https://github.com/unocss/unocss/assets/104117864/5e940dc5-c666-4f7e-97db-05586d62948a)
但自定义的样式却可以用
![image](https://github.com/unocss/unocss/assets/104117864/d51968ed-b311-4fdc-82e0-7e609773e27e)
如下是相关的uno-config的配置
![image](https://github.com/unocss/unocss/assets/104117864/8374efd0-2dcb-4e2b-88c1-1b9517782c85)
main.ts和vite.config里也都按文档配置了
![image](https://github.com/unocss/unocss/assets/104117864/14e8b23c-2dd6-4944-ae6b-6aab01caba63)
![image](https://github.com/unocss/unocss/assets/104117864/359529cd-70c8-491e-b064-df10cbbff3eb)

**Comment by zyyv** [comment, marked as answer]
没有引用 `presetUno`

**Reply by rabbit-lili** [reply]
当局者迷，非常感谢
